easybuild/tools/options.py

```diff
@@ -562,6 +562,10 @@ def github_options(self):
             'github-user': ("GitHub username", str, 'store', None),
             'github-org': ("GitHub organization", str, 'store', None),
             'install-github-token': ("Install GitHub token (requires --github-user)", None, 'store_true', False),
+            'list-prs': ("List pull requests", None, 'store_true', None),
+            'list-prs-state': ("List pull requests in state open/closed", str, 'store', 'open'),
```

Comment:
Make this a `choice` option, anything other than `open` or `close` doesn't make sense?